Sources/HarmonizeSemantics/Declaration/Initializer.swift

```diff
@@ -47,7 +48,11 @@ extension Initializer: AttributesProviding,
     public var body: String? {
         node.body?.statements.toString()
     }
-    
+
+    public var statements: [String] {
+        node.body?.statements.map { $0.trimmedDescription } ?? []
+    }
```

Comment:
@lcszc do we need a test here?

Reply:
Yes.

But what do you think about creating a model for Body? As we now have statements (and we may have more properties in the future), we could put this into `BodyProviding`.

Even better, if we create a `Body` as a grammar component declaration we can do:

```swift
struct Body:  DeclarationDecoration {
  public let node: CodeBlockSyntax
  
  // ...
  
  var text: String
  
  var lines: String // or statements
}
```

What do you think? If that's too much I'd just recommend to move `statements` to `BodyProviding` and also to add your `statements` implementation for functions as well.